tests/ballerina-tools-integration-test/src/test/resources/packaging/ballerinaSearch.out

```diff
@@ -0,0 +1,8 @@
+
+Ballerina Central
+=================
+
+|NAME                                                  | DESCRIPTION                                                                       | AUTHOR         | DATE           | VERSION |
+|------------------------------------------------------| ----------------------------------------------------------------------------------| ---------------| ---------------| --------|
+|natasha/searchTestPackage                             | Prints "hello world" to command line output                                       |                | 2018-07-06-Fri | 1.0.0   |
```

Comment:
This can change depending on the user's screen size right ?

Reply:
In this case the terminal width will be always be 80